be/src/olap/tablet_schema.h

```diff
@@ -363,17 +366,17 @@ class TabletSchema {
     // Dump [(name, type, is_nullable), ...]
     string dump_structure() const {
         string str = "[";
-        for (auto p : _field_name_to_index) {
+        for (auto p : _cols) {
```

Comment:
why change this code?

Reply:
`_field_name_to_index` is an unorded_map. change it to make output ordered